examples/scenes/src/lib.rs

```diff
@@ -46,6 +47,12 @@ pub struct Arguments {
     #[arg(help_heading = "Scene Selection", global(false))]
     /// The svg files paths to render
     svgs: Option<Vec<PathBuf>>,
+    #[arg(help_heading = "Render Parameters")]
+    #[arg(long, global(false))]
+    /// The base color applied as the blend background to the rasterizer.
+    /// Format is CSS style hexidecimal (#RGB, #RGBA, #RRGGBB, #RRGGBBAA) or
```

Comment:
The one issue with this is that in powershell, `#` starts a comment. For example:
```powershell
cargo run ... --base-color #bed
```
needs to be written as (with the `` ` `` as the escape character)
```powershell
cargo run ... --base-color `#bed
```

That's not really the fault of this PR though. I don't think we need to add a comment here

Reply:
I would agree that this is a shell-specific concern.